speclang_modern.tex

```diff
@@ -2305,7 +2305,12 @@ \section{Pointers and physical addressing}
 The grammar for terms and predicates is
 extended with new constructs given in Figure~\ref{fig:gram:memory}. 
 \textsl{Location-address} denotes the address of a memory location. 
-It is a set of terms of some pointer type as defined in Section~\ref{sec:locations}.
+It is a set of terms of some pointer type as defined in Section~\ref{sec:locations}. As indicated below where necessary,
+many built-in functions and predicates dealing with
+pointers depend on the size of the referenced type. Thus, they cannot be given
+as argument a pointer to \lstinline|void|. On the
+other hand, a pointer referencing an incomplete type (hence having an
+abstract size) is possible.
```

Comment:
@pbaudin I've added a small paragraph on `void *` vs pointer referencing incomplete type. Is that OK with you?

Reply:
perfect.